incubator/azuremonitor-containers/README.md

```diff
@@ -29,11 +31,12 @@ This article describes how to set up and use [Azure Monitor - Containers](https:
 
 ### To Use Azure Log Analytics Workspace in Public Cloud
 
+> Note: If you are using Helm3 client, use `--generate-name` instead of `--name myrelease-1` for the install command
```

Comment:
This isn't necessary

Reply:
> This isn't necessary

revised Note to cover the deprecated flag.